templates/ai/mlops.html

```diff
@@ -20,6 +20,7 @@
     <div class="p-section--deep">
       <div class="row">
         <div class="col-start-large-4 col-9">
+          {% include "shared/_azure-banner.html" %}
```

Comment:
I'm not sure if this is supposed to start 4 cols in or if it's supposed to span the full row. The [mock-up](https://www.figma.com/design/sGeCjFvA1kV9Kmvg2S2VBn/23.10-U.com---AI-bubble?node-id=248-1446&node-type=canvas&t=Uorg5aCnmz3Gf6it-0) provided doesn't seem to match the current page, but the version that has the existing suru has the navigation spanning the full width. @mattea-turic could you help here?

Reply:
@mtruj013 

Am I understanding the question correctly, in that you're not sure whether the banner in the mockup or the demo is correct in design? Since this page looks to be one of the early rebranded ones, I'd keep it as is, starting 4 cols in. 

Ideally, the entire content would start at the first column like all the newer pages, so maybe we can refresh this? Then it'll be consistent with the other pages in the bubble, aside from consulting.

Reply:
@mattea-turic , since this change is being considered as time sensitive, I'd not align that page with the new design. Would that work for you?

Reply:
@yurii-vasyliev Yes, that's works